synapse/lib/stormtypes.py

```diff
@@ -6009,6 +6022,45 @@ async def _libVarsList(self):
     async def _libVarsType(self, valu):
         return await totype(valu)
 
+@registry.registerType
+class PkgVars(Prim):
+    '''
+    The Storm deref/setitem/iter convention on top of pkg vars information.
+    '''
+    _storm_typename = 'pkg:vars'
+    _ismutable = True
+
+    def __init__(self, runt, valu, path=None):
+        Prim.__init__(self, valu, path=path)
+        self.runt = runt
+
+    def _reqPkgAdmin(self):
+        self.runt.confirm(('power-ups', self.valu, 'admin'))
```

Comment:
We need to start documenting this permission hierarchy as part of the permdefs in `LibPkg`.

Reply:
added